tools/formatter/FormatterTool.java

```diff
@@ -96,6 +96,19 @@ public void format(Path path) throws IOException {
     }
   }
 
+  static class BuildFormatter implements BaseFormatter {
+    static BuildFormatter init() {
+      return new BuildFormatter();
+    }
+
+    @Override
+    public void format(Path path) throws IOException {
+      String command =
+          "/usr/bin/env bash " + "tools/buildtools_wrappers/buildifier.sh -mode=fix " + "%s";
```

Comment:
Does this work on both Linux and Mac?

Reply:
@oferb yes, because script picks the correct binary